### PR TITLE
Updating the member page style according to the Figma design

### PR DESCRIPTION
**Pull request by develad**
Hey everyone!

I've updated the member page style according to the Figma design and updated the images resource address, in the current published version the member image can't be loaded and there's a console error. 

Also I left the font Birzia ( I think it's looking cooler ) and left the Discord link button on mobile view.

Check out the screen shots attached:  

localhost_3001_he_members (1)

localhost_3001_he_members

localhost_3001_he_members(iPhone 14 Pro Max) (1)

### CODE REVIEW

File: app/[locale]/members/page.tsx [hunk 1]
```diff
@@ -9,66 +9,77 @@ import { useState } from 'react';
...
-      <div className="flex justify-evenly flex-wrap">
-        <p className="text-2xl">
+    <div className="flex flex-col justify-center bg-purple-100 dark:bg-gray-800 mb-6 mt-2 md:mb-12 py-8 px-4 md:p-4">
+      <div className="flex flex-col md:flex-row justify-evenly">
+        <p className="text-2xl text-center md:text-right">
           כאן אפשר להכיר את כל המשפחה! וכשתסיימו, בואו לדיסקורד
         </p>
-        <div>
+        <div className="">
           <DiscordLink
```
Comment (Tamir198) [inline app/[locale]/members/page.tsx:80]: Why the empty class name

File: components/Members/MemberCard/MemberCardInfo.tsx [hunk 2]
```diff
@@ -15,27 +33,37 @@ export const MemberCardInfo = ({
...
         />
 
         <div className="titles-container block">
-          <p className="text-base">{name}</p>
-          <p className="text-sm text-slate-500 dark:text-slate-300">
+          <div className="flex items-center">
+            <p className="text-base ml-4">{name}</p>
+            {isAdmin && (
+              <Crown className="w-4 h-4 fill-[#4338CA] dark:fill-[#DBC927]" />
+            )}
```
Comment (Tamir198) [inline components/Members/MemberCard/MemberCardInfo.tsx:54]: How about we export this into a separate component? seems like something that can be used in the future across the website

**Review by Tamir198 — APPROVED** [review]

**Comment by Tamir198** [comment]
Also, I thought it could be a cool feature to have a fallback image in case some user will lack a profile image.
what do you think?

File: components/Members/MemberCard/MemberCardInfo.tsx (hunk 2 above)
Comment (develad) [reply components/Members/MemberCard/MemberCardInfo.tsx:54]: @Tamir198 
> How about we export this into a separate component? seems like something that can be used in the future across the website

Fine by me :) you can export this into a separate component

**Comment by develad** [comment]
@Tamir198 
> Also, I thought it could be a cool feature to have a fallback image in case some user will lack a profile image. what do you think?

Yes! It's a great idea. can you implement this feature? if you work on my files please pull the latest changes from my 'member-page' branch

**Comment by Tamir198** [comment]
I don`t mind working on it but lets wait and merge this pr first to avoid conflicts

**Comment by develad** [comment]
@Darkmift 
Thank you for the merge! I'm happy to contribute to this on going project

**Pull request merged**